event/event.go

```diff
@@ -243,6 +245,26 @@ func GetTargetType(t string) (TargetType, error) {
 		return TargetTypeTeam, nil
 	case "user":
 		return TargetTypeUser, nil
+	case "iaas":
```

Comment:
nit: I think we could refactor this function to use a map string -> TargetType

Reply:
Yeah, ugly. I think this should be either generated or removed, I'm not sure if we really need this GetTargetType anymore. I'll address this in another PR.

Reply:
👍